cores/arduino/stm32/usb/usbd_conf.c

```diff
@@ -51,6 +51,13 @@ PCD_HandleTypeDef g_hpcd;
 void HAL_PCD_MspInit(PCD_HandleTypeDef *hpcd)
 {
   const PinMap *map = NULL;
+
+#if defined(PIN_UCPD_TCPP)
+  /* Set TCPP default state: Type-C legacy */
+  pin_function(digitalPinToPinName(PIN_UCPD_TCPP), STM_PIN_DATA(STM_MODE_OUTPUT_OD, GPIO_NOPULL, 0));
```

Comment:
Why not simply use `pinMode(PIN_UCPD_TCPP, OUTPUT_OPEN_DRAIN);`?
Require to be faster ?

Reply:
Done